src/comments/comments.controller.ts

```diff
@@ -0,0 +1,74 @@
+import { Body, Controller, Delete, Get, Param, ParseIntPipe, Patch, Post, Query, Req, UseGuards } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { Request } from 'express';
+import { CommentsService } from './comments.service';
+import { CreateCommentDto } from './dto/create-comment.dto';
+import { UpdateCommentDto } from './dto/update-comment.dto';
+
+@Controller('/posts')
+export class CommentsController {
+  constructor(private readonly commentsService: CommentsService) {}
+
+  @UseGuards(AuthGuard('jwt'))
+  @Post(':postId/comments')
+  async createComment(
+    @Param('postId', ParseIntPipe) postId: number,
+    @Body() createCommentDto: CreateCommentDto,
+    @Req() req: Request,
+  ) {
+    const user = req.user as { userId: number };
+    const userId = user.userId;
```

Comment:
```suggestion
    const { userId } = req.user as { userId: number };
```

이렇게도 표현 가능할거같아요 물론 create 함수에 user.id로 넘겨줘도 무방합니다